### ISSUE TITLE
ngscheckmate

### ISSUE THREAD

**Issue by ewels** [nf-core/ops#76]

**Comment by ewels** [comment]
> dependency issues - darn thing wants python 2.7, which causing fails in the dependencies.

**Comment by SPPearce** [comment]
> > dependency issues - darn thing wants python 2.7, which causing fails in the dependencies.

Yeah, this is PhD-abandonware I'm afraid. I did manage to get a PR merged last year by hassling one of the authors.

**Comment by dslarm** [comment]
There's only ~4k lines of python - and python 2 -> 3 is just a bit of tab->spaces and putting brackets round print statements (if we're very lucky).
I'm giving it a try.

**Comment by dslarm** [comment]
https://github.com/bioconda/bioconda-recipes/pull/53196
